tests/integration/records/serializers/test_json.py

```diff
@@ -287,9 +291,11 @@ def test_literature_json_ui_v1_response(api_client, db, create_record):
     response_status_code = response.status_code
     response_data = json.loads(response.data)
     response_data_metadata = response_data["metadata"]
+    response_data_uuid = response_data["uuid"]
 
     assert expected_status_code == response_status_code
     assert expected_result_metadata == response_data_metadata
+    assert expected_uuid == response_data_uuid
```

Comment:
could we also add the ``id``, ``created``, ``updated`` as well?  I think they are missing from the assertions.